### ISSUE TITLE
Update HAL Link "Part Of" to include links for both directions in Work records

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-middletier#79]
**Problem Description:**
The current "Part of" Hal link renders data for works Part Of another work. However it does not include the opposite direction of linking the work `part of` another work back to the original work record. 

**Expected Behavior/Solution:**
Update the HAL link to go in both directions so work records which are `part of` another work can go back to the original work. 

**Requirements:**
- ML Query

**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- [ ] Wireframe/Mockup - Mike
- [ ] Committee discussions - Sarah
- [ ] Feasibility/Team discussion - Sarah
- [ ] Backend requirements - TBD
- [ ] Frontend requirements- TBD
- [ ] Are new regression tests required for QA - Amy
- [ ] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**
*Link one needs to link back to link two as link two links back to link one. 
-https://lux-front-dev.collections.yale.edu/view/text/66b917f7-6908-43aa-8e0e-5d2f4b26424c 
- https://lux-front-dev.collections.yale.edu/view/text/77a59f7f-227f-4e8d-8539-39fc39a15b3e

**Dependencies/Blocks:**

- Blocked By: Issues that are blocking the completion of the current issue.
- Blocking: Issues being blocked by the completion of the current issue.

**Related Github Issues:**
- https://github.com/project-lux/data-pipeline/issues/85

**Related links:**
- Bugherd: https://www.bugherd.com/projects/284041/tasks/2327

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by kkdavis14** [comment]
@clarkepeterf @kamerynB this might be going away/a non-issue. Stay tuned.

**Comment by roamye** [comment]
@kkdavis14 - Does your comment apply to the ML and FE tickets related to this? I assume so but wanted to clarify. 

(https://github.com/project-lux/lux-frontend/issues/302 & https://github.com/project-lux/lux-marklogic/issues/234)

**Comment by kkdavis14** [comment]
@roamye yes, it would apply to everything--but still confirming the data. might take a few weeks.

**Comment by kkdavis14** [comment]
@roamye @clarkepeterf @kamerynB ok, sorry about that--we DO want to move forward on this ticket. We thought it was all potentially bad data and didn't need accounted for, but there's enough records affected (380k) to implement this. 
still blocked by needing the triple.

**Comment by roamye** [comment]
Approved by UAT

**Comment by roamye** [comment]
Closing as the FE looks good in PROD.
